woocommerce/woocommerce
# Removed 100% width on images on single product page for 2017 theme

### All Submissions:






### Changes proposed in this Pull Request:



Closes #20797  .

### How to test the changes in this Pull Request:

1. Add image having width < 200px to a product.
2. Check single product page using Twenty Seventeen theme, see the image is stretched to 100% width.
3. Switch to RTL to check the image displays correctly.
3. Apply branch
4. Observe the image is no longer stretched.
3. Switch to RTL to check the image displays correctly.


### Other information:




### Changelog entry

> Removed 100% width for images on single product page.
